### DISCUSSION TITLE
Tbdb.bed vs genome.gff coordinates question

### DISCUSSION THREAD

**Discussion by taranewman** [jodyphelan/TBProfiler#398] [Q&A] [answered]
Hi Jody,

I just wanted to clarify the positions in the https://github.com/jodyphelan/TBProfiler/blob/master/db/tbdb.bed file vs the https://github.com/jodyphelan/TBProfiler/blob/master/db/tbdb.gff file

As an example: 

Rv2752c is listed as having an end position  3066191 in the tbdb.gff 

Chromosome	ena	gene	3064515	3066191	.	-	.	ID=gene:Rv2752c;

In the tbdb.bed it is listed as having an end position of 3067372 (but the same start position)

Chromosome	3064515	3067372	Rv2752c	Rv2752c	levofloxacin,moxifloxacin,isoniazid,rifampicin,ethambutol

Is the difference in end position related to upstream variants that may impact gene resistance?

Another example is Rv0010c

Thank you for your time,
Tara

**Comment by jodyphelan** [comment, marked as answer]
Hi Tara,

Sorry for the delay in getting back to you. Yes the start position in the bed file contains the upstream region where potential promoter mutations may be.

**Reply by taranewman** [reply]
Thanks Jody!
